bin/test-all-examples

```diff
@@ -9,6 +9,22 @@ fi
 
 exercisedir=$1
 
+if [ "$current_branch" != "master" ]; then
```

Comment:
I did not modify `test-examples`, only `test-all-examples`. The reasoning being if an exercise is not modified, we want to skip all examples. Thus, testing individually in `test-examples` can be "lifted" into `test-all-examples`.

I hope I didn't misunderstand the purpose of the examples here, though.

Reply:
While I did somehow misunderstand and thought that `test-example` had been used, this now leaves an interesting question. Since `test-example` is not changed, that means if someone were to choose to directly run `test-example` on one individual example, it would not benefit from this addition.

But I suppose that's rare enough that we don't need to think about it. In that case, I have no further change to suggest in this vein.

Reply:
Are these scripts meant to be run manually? In this case, this would require the addition of something like a `--force` switch.

If they're only used by travis, I don't think it's a big problem, as running a `test-example` explicitly indicates strongly that you'd want those examples tested.

Reply:
Ah, but this would require all the travis environment variables. I'll make the script runnable in manual mode as well.